src/api/index.js

```diff
@@ -154,6 +157,23 @@ class Steem extends EventEmitter {
             .then(res => { callback(null, res) }, err => { callback(err) });
     }
 
+    signedCall(method, params, account, key, callback) {
```

Comment:
make this `async` and return the promise from `jsonRpc`

Reply:
I thought the api methods had some thing iterating over them converting callbacks to promises

Reply:
Currently only the steemd methods are promisified on not every methods of `steem.api` https://github.com/steemit/steem-js/blob/11ecd391a971133fca657f55aafb6e26bf4bbe2a/src/api/index.js#L49-L50

Reply:
yeah, make sure this method talks promises